01-spring-boot/README.md

```diff
@@ -6,13 +6,49 @@ We are going to create a project using Spring Boot. Take a moment to think about
 * Copy and pasted from StackOverflow until something worked?
 * Found a project within an organisation and used the same thing (just renaming a few things)
 
-Our workshop starts with a empty folder. We are going to use [SpringInitalizr](https://start.spring.io) to create our project and get started.
+Our workshop starts with a empty folder. We are going to use [SpringInitalizr](<https://start.spring.io>) to create our project and get started.
 
-If you have not completed the [pre-requisites](../prerequisites/README.md) please complete these steps first.
+## Step 1 - Generating the Project
 
-### Step 1 - Creating our First Controller
+If you've already completed the [pre-requisites](../prerequisites/README.md) you can skip forward to Step 3.
 
-In the project we will create a small REST controller. 
+* Visit [https://start.spring.io](<https://start.spring.io>) to create a new project
```

Comment:
Something looks to have gone wrong here. Perhaps you are working on an old version. We have removed the steps from this lab to point only to the pre-reqs to avoid duplication

Reply:
Added changes. It was because I modified some of the lines of a section it seems to have got confused when I merged